plugin/fzf.vim

```diff
@@ -484,7 +487,8 @@ try
     elseif type == 3
       let temps.input = s:fzf_tempname()
       call s:writefile(source, temps.input)
-      let source_command = (s:is_win ? 'type ' : 'cat ').fzf#shellescape(temps.input)
+      let source_command = (s:is_win && !s:is_gitbash ? 'type ' : 'cat ')
```

Comment:
If I'm not mistaken, `environ()` is a recent addition to Vim (after Vim 8), so it's probably safer to use `exists('$VARNAME')` not to break this on older Vim versions.

```suggestion
      let source_command = (s:is_win && !exists('$SHELL') ? 'type ' : 'cat ')
```